recipes/pthreads4w/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class Pthreads4WConan(ConanFile):
+    name = "pthreads4w"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://sourceforge.net/projects/pthreads4w/"
+    description = "POSIX Threads for Windows"
+    license = "LGPL-2.0"
```

Comment:
```suggestion
    license = "Apache-2.0"
```

According the file LICENSE in the project, it's Apache 2.0

Reply:
indeed, it's apache since v3.0.0. thanks !